lgc/util/StartStopTimer.cpp

```diff
@@ -58,7 +63,23 @@ class StartStopTimer : public ModulePass {
   bool m_starting; // True to start the timer, false to stop it
 };
 
-char StartStopTimer::ID = 0;
+// Legacy PM wrapper
```

Comment:
Please add the separate line. Each class declaration needs a separate line and comments describing its function. Please reference existing codes.
Also, PM -> pass manager